docs/docs/getting-started/docker-sidecar.md

```diff
@@ -0,0 +1,51 @@
+# Create a Docker Sidecar Function
+
+Lug Docker Sidecar functions are Lug Hybrid functions with an extra benefit: they attach a running Docker container to 
+your function. This is particularly useful for programs with very specific requirements function best in their own 
+separate container.
```

Comment:
Think there's a missing "that" or similar in this last sentence.

Reply:
Fixed